nasa/harmony
# Harmony 790 expiration dates

## Jira Issue ID
HARMONY-790

## Description
Adds expiration date to job status page and STAC items

## Local Test Steps
* Issue an async Harmony request
* Go to the job status page and verify that the status includes a `dataExpiration` field that is 30 days in the future
* Append a '/0' to the end of the STAC catalog link to get the first STAC item
* Verify that the STAC item contains a a `stac_extensions` field with the url of the 'timestamps' extension and that it contains a `exprires` field in the `properties` section.

## PR Acceptance Checklist
